packages/nc-gui/components/project/View.vue

```diff
@@ -119,14 +148,14 @@ watch(
               <GeneralIcon icon="users" class="!h-3.5 !w-3.5" />
               <div>{{ $t('labels.members') }}</div>
               <div
-                v-if="workspaceUserCount"
+                v-if="isEeUI ? workspaceUserCount : baseUserCount"
```

Comment:
L151 & L158 are same - define once and use multiple times instead.

Reply:
Updated